src/GitVersionCore/VersionCalculation/BaseVersionCalculator.cs

```diff
@@ -28,23 +28,27 @@ public BaseVersion GetBaseVersion()
             using (log.IndentLog("Calculating base versions"))
             {
                 var baseVersions = strategies
-                    .SelectMany(s => s.GetVersions())
-                    .Where(v =>
+                    .SelectMany(s =>
                     {
-                        if (v == null) return false;
-
-                        log.Info(v.ToString());
-
-                        foreach (var filter in context.Configuration.VersionFilters)
-                        {
-                            if (filter.Exclude(v, out var reason))
+                        return s.GetVersions()
+                            .Where(v =>
                             {
-                                log.Info(reason);
-                                return false;
-                            }
-                        }
-
-                        return true;
+                                if (v == null) return false;
+                                if (s is FallbackVersionStrategy) return true;
```

Comment:
Couldn't this this `if` statement be moved to the outer scope since it only concerns itself with the strategies `s` variable?

Reply:
Good point, updated and pushed.